paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -2260,7 +2260,8 @@ PDNode *patterns::OpRequant::operator()() {
   auto any_op = pattern->NewNode(any_op_repr())
                     ->assert_is_op()
                     ->assert_more([&](Node *node) {
-                      return node->Op()->HasAttr("Scale_out") ? true : false;
+                      return (node->Op()->HasAttr("Scale_out") ||
```

Comment:
Can `Scale_out` be deleted?

Reply:
There isn't a consistent naming scheme and many operators have "Scale" and many others have "scale", so it can be deleted once all operators have the same capitalization scheme